.pre-commit-config.yaml

```diff
@@ -60,6 +60,7 @@ repos:
       - id: isort
         name: isort (python)
         files: ^api/.*\.py$
+        args: ["--diff"]
```

Comment:
why?

Reply:
should not be included. Had some issues with isort in CI. Fixed now.